### ISSUE TITLE
regenerate pod file problem 

### ISSUE THREAD

**Issue by roshaninet** [flutter/flutter#71216]
hi 
i have seen this warning on my fultter app ios building: 

```
Warning: Podfile is out of date
  This can cause issues if your application depends on plugins that do not support iOS.
  See https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms for details.
  If you have local Podfile edits you would like to keep, see https://github.com/flutter/flutter/issues/45197 for instructions.
To regenerate the Podfile, run:
  rm ios/Podfile
```

and i did `rm ios/Podfile`

my new pod file created by flutter: 
Some basic Git commands are:
```
platform :ios, '10.0'

ENV['COCOAPODS_DISABLE_STATS'] = 'true'

project 'Runner', {
  'Debug' => :debug,
  'Profile' => :release,
  'Release' => :release,
}

def flutter_root
  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
  unless File.exist?(generated_xcode_build_settings_path)
    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
  end

  File.foreach(generated_xcode_build_settings_path) do |line|
    matches = line.match(/FLUTTER_ROOT\=(.*)/)
    return matches[1].strip if matches
  end
  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
end

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)

flutter_ios_podfile_setup

target 'Runner' do
  use_frameworks!
  use_modular_headers!

  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
end
```
now i have another problem: 

```
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1462:38: warning:
    variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1426:3: note: variable
    'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1462:48: warning:
    variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1425:3: note: variable
    'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1546:38: warning:
    variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1510:3: note: variable
    'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1546:48: warning:
    variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1509:3: note: variable
    'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    4 warnings generated.
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/enc/backward_references_enc.c:663:12:
    warning: variable 'best_offset' may be uninitialized when used here [-Wconditional-uninitialized]
              (best_offset << MAX_LENGTH_BITS) | (uint32_t)best_length;
               ^~~~~~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/enc/backward_references_enc.c:595:20:
    note: initialize the variable 'best_offset' to silence this warning
        int best_offset;
                       ^
                        = 0
    1 warning generated.
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1462:38: warning:
    variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1426:3: note: variable
    'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1462:48: warning:
    variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1425:3: note: variable
    'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1546:38: warning:
    variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1510:3: note: variable
    'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1546:48: warning:
    variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/dsp/dec_neon.c:1509:3: note: variable
    'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    4 warnings generated.
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/enc/backward_references_enc.c:663:12:
    warning: variable 'best_offset' may be uninitialized when used here [-Wconditional-uninitialized]
              (best_offset << MAX_LENGTH_BITS) | (uint32_t)best_length;
               ^~~~~~~~~~~
    /Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/ios/Pods/libwebp/src/enc/backward_references_enc.c:595:20:
    note: initialize the variable 'best_offset' to silence this warning
        int best_offset;
                       ^
                        = 0
    1 warning generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:35:65: warning:
    'kCGImagePropertyTIFFTileWidth' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                 SYStringSel(tileWidth):                (NSString *)kCGImagePropertyTIFFTileWidth,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ImageIO' imported from /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:10:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.2.sdk/System/Library/Frameworks/
    ImageIO.framework/Headers/CGImageProperties.h:149:34: note: 'kCGImagePropertyTIFFTileWidth' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyTIFFTileWidth  IMAGEIO_AVAILABLE_STARTING(10.11, 9.0);
                                     ^
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:35:65: note:
    enclose 'kCGImagePropertyTIFFTileWidth' in an @available check to silence this warning
                 SYStringSel(tileWidth):                (NSString *)kCGImagePropertyTIFFTileWidth,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:36:65: warning:
    'kCGImagePropertyTIFFTileLength' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                 SYStringSel(tileLength):               (NSString *)kCGImagePropertyTIFFTileLength,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ImageIO' imported from /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:10:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.2.sdk/System/Library/Frameworks/
    ImageIO.framework/Headers/CGImageProperties.h:150:34: note: 'kCGImagePropertyTIFFTileLength' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyTIFFTileLength  IMAGEIO_AVAILABLE_STARTING(10.11, 9.0);
                                     ^
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:36:65: note:
    enclose 'kCGImagePropertyTIFFTileLength' in an @available check to silence this warning
                 SYStringSel(tileLength):               (NSString *)kCGImagePropertyTIFFTileLength,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 warnings generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataPNG.m:32:61: warning:
    'kCGImagePropertyPNGCompressionFilter' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                 SYStringSel(compressionFilter):    (NSString *)kCGImagePropertyPNGCompressionFilter,
                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ImageIO' imported from /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataPNG.h:10:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.2.sdk/System/Library/Frameworks/
    ImageIO.framework/Headers/CGImageProperties.h:784:34: note: 'kCGImagePropertyPNGCompressionFilter' has been marked as being
    introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyPNGCompressionFilter IMAGEIO_AVAILABLE_STARTING(10.11, 9.0);
                                     ^
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataPNG.m:32:61: note:
    enclose 'kCGImagePropertyPNGCompressionFilter' in an @available check to silence this warning
                 SYStringSel(compressionFilter):    (NSString *)kCGImagePropertyPNGCompressionFilter,
                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 warning generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataExif.m:9:9: warning:
    non-portable path to file '"SYMetadataExif.h"'; specified path differs in case from file name on disk
    [-Wnonportable-include-path]
    #import "SYMetadataEXIF.h"
            ^~~~~~~~~~~~~~~~~~
            "SYMetadataExif.h"
    1 warning generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/FlutterImageCompressPlugin.m:32:30: warning:
    incompatible pointer to integer conversion sending 'id _Nullable' to parameter of type 'BOOL' (aka 'signed char')
    [-Wint-conversion]
                [self setShowLog:[call arguments]];
                                 ^~~~~~~~~~~~~~~~
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/FlutterImageCompressPlugin.m:48:26: note: passing
    argument to parameter 'log' here
    - (void)setShowLog:(BOOL)log{
                             ^
    1 warning generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:35:65: warning:
    'kCGImagePropertyTIFFTileWidth' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                 SYStringSel(tileWidth):                (NSString *)kCGImagePropertyTIFFTileWidth,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ImageIO' imported from /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:10:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.2.sdk/System/Library/Frameworks/
    ImageIO.framework/Headers/CGImageProperties.h:149:34: note: 'kCGImagePropertyTIFFTileWidth' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyTIFFTileWidth  IMAGEIO_AVAILABLE_STARTING(10.11, 9.0);
                                     ^
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:35:65: note:
    enclose 'kCGImagePropertyTIFFTileWidth' in an @available check to silence this warning
                 SYStringSel(tileWidth):                (NSString *)kCGImagePropertyTIFFTileWidth,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:36:65: warning:
    'kCGImagePropertyTIFFTileLength' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                 SYStringSel(tileLength):               (NSString *)kCGImagePropertyTIFFTileLength,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ImageIO' imported from /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:10:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.2.sdk/System/Library/Frameworks/
    ImageIO.framework/Headers/CGImageProperties.h:150:34: note: 'kCGImagePropertyTIFFTileLength' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyTIFFTileLength  IMAGEIO_AVAILABLE_STARTING(10.11, 9.0);
                                     ^
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataTIFF.m:36:65: note:
    enclose 'kCGImagePropertyTIFFTileLength' in an @available check to silence this warning
                 SYStringSel(tileLength):               (NSString *)kCGImagePropertyTIFFTileLength,
                                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 warnings generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataPNG.m:32:61: warning:
    'kCGImagePropertyPNGCompressionFilter' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                 SYStringSel(compressionFilter):    (NSString *)kCGImagePropertyPNGCompressionFilter,
                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ImageIO' imported from /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataPNG.h:10:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.2.sdk/System/Library/Frameworks/
    ImageIO.framework/Headers/CGImageProperties.h:784:34: note: 'kCGImagePropertyPNGCompressionFilter' has been marked as being
    introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyPNGCompressionFilter IMAGEIO_AVAILABLE_STARTING(10.11, 9.0);
                                     ^
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataPNG.m:32:61: note:
    enclose 'kCGImagePropertyPNGCompressionFilter' in an @available check to silence this warning
                 SYStringSel(compressionFilter):    (NSString *)kCGImagePropertyPNGCompressionFilter,
                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 warning generated.
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataExif.m:9:9: warning:
    non-portable path to file '"SYMetadataExif.h"'; specified path differs in case from file name on disk
    [-Wnonportable-include-path]
    #import "SYMetadataEXIF.h"
            ^~~~~~~~~~~~~~~~~~
            "SYMetadataExif.h"
    1 warning generated.
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImageWebPCoder/SDWebImageWeb
    PCoder.framework/SDWebImageWebPCoder) was built for newer iOS version (9.0) than being linked (8.0)
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImage/SDWebImage.framework/S
    DWebImage) was built for newer iOS version (9.0) than being linked (8.0)
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImageWebPCoder/SDWebImageWeb
    PCoder.framework/SDWebImageWebPCoder) was built for newer iOS version (9.0) than being linked (8.0)
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImage/SDWebImage.framework/S
    DWebImage) was built for newer iOS version (9.0) than being linked (8.0)
    ld: framework not found flutter_plugin_android_lifecycle
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    ld: framework not found flutter_plugin_android_lifecycle
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'BSGridCollectionViewLayout' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'url_launcher' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'libwebp' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'webview_flutter' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'share' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'image_picker' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'shared_preferences' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'path_provider' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'google_maps_flutter' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'flutter_secure_storage' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'flutter_image_compress' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'Mantle' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'BSImageView' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'Flutter' from project 'Pods')

Encountered error while building for device.
```

**Comment by darshankawar** [comment]
@roshaninet 
Can you try following commands and see if it works:

```
flutter clean 
rm ios/Podfile
rm ios/Podfile.lock 
rm pubspec.lock 
rm -rf ios/Pods ios/Runner.xcworkspace 
flutter run
```
It seems the `flutter_plugin_android_lifecycle` isn't removed / updated after you ran `rm ios/podfile`.
If issue persists, please provide `pubspec.yaml`.

**Comment by roshaninet** [comment]
@darshankawar problem still exist ! 

```

Xcode's output:
↳
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/FlutterImageCompressPlugin.m:32:30: warning:
    incompatible pointer to integer conversion sending 'id _Nullable' to parameter of type 'BOOL' (aka 'signed char')
    [-Wint-conversion]
                [self setShowLog:[call arguments]];
                                 ^~~~~~~~~~~~~~~~
    /Users/mohammadroshani/flutersdk/flutter
    2/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/FlutterImageCompressPlugin.m:48:26: note: passing
    argument to parameter 'log' here
    - (void)setShowLog:(BOOL)log{
                             ^
    1 warning generated.
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImageWebPCoder/SDWebImageWeb
    PCoder.framework/SDWebImageWebPCoder) was built for newer iOS version (9.0) than being linked (8.0)
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImage/SDWebImage.framework/S
    DWebImage) was built for newer iOS version (9.0) than being linked (8.0)
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImageWebPCoder/SDWebImageWeb
    PCoder.framework/SDWebImageWebPCoder) was built for newer iOS version (9.0) than being linked (8.0)
    ld: warning: dylib
    (/Users/mohammadroshani/Projects/shabesh_flutter/shabesh_application/build/ios/Release-iphoneos/SDWebImage/SDWebImage.framework/S
    DWebImage) was built for newer iOS version (9.0) than being linked (8.0)
    ld: framework not found flutter_plugin_android_lifecycle
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'webview_flutter' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'google_maps_flutter' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'BSImageView' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'Mantle' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'image_picker' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'share' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'flutter_secure_storage' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'shared_preferences' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'url_launcher' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'path_provider' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'libwebp' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'BSGridCollectionViewLayout' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'flutter_image_compress' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 14.2.99. (in target 'Flutter' from project 'Pods')

Encountered error while building for device.
```

**Comment by darshankawar** [comment]
@roshaninet 
Can you try [this](https://github.com/flutter/flutter/issues/63220#issuecomment-702055401) possible solution and see if it works ?

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
